ui/App/ProjectScreen/Source/Anchors.svelte

```diff
@@ -0,0 +1,223 @@
+<!--
+ Copyright © 2021 The Radicle Upstream Contributors
+
+ This file is part of radicle-upstream, distributed under the GPLv3
+ with Radicle Linking Exception. For full terms see the included
+ LICENSE file.
+-->
+<script lang="typescript">
+  import type * as project from "ui/src/project";
+
+  import * as zod from "zod";
+
+  import * as browserStore from "ui/src/browserStore";
+  import * as ethereum from "ui/src/ethereum";
+  import * as org from "ui/src/org";
+  import * as router from "ui/src/router";
+
+  import { Icon, IdentifierLink } from "ui/DesignSystem";
+  import EmptyState from "ui/App/ScreenLayout/EmptyState.svelte";
+  import OrgIdentity from "ui/App/OrgIdentity.svelte";
+
+  export let anchors: project.ConfirmedAnchor[];
+
+  const latest = anchors.slice(-1)[0];
+  const oldAnchors = anchors.slice(0, -1);
```

Comment:
To ensure this is reactive

```suggestion
  $: latest = anchors.slice(-1)[0];
  $: oldAnchors = anchors.slice(0, -1);
```

Reply:
1066991c037263586762180e95e64f9e59b89cda